doc/content/lorawan/lorawan-relay/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: LoRaWAN Relay
+section: The Things Fundamentals
+weight: 50
+images:
+- relay-placement.png
+---
+
+Relays are suitable for bridging wireless communication gaps in areas where the gateway and end device cannot directly communicate with each other due to weak signal strength because of factors such as extreme distance or obstacles between the gateway and end device. They are low-cost, low-power devices. A relay's hardware is equivalent to a standard LoRaWAN end device. A single relay can serve up to 16 end devices. For the Network Server, a relay appears as a standard end device.
+
+The following figure shows how a relay helps to forward/relay a message from an end device to the gateway.
+
+{{< figure src="relay-placement.png" alt="Relay Placement" >}}
+
+{{< note "Adding a new gateway is a good idea if many devices have a lack of signal reception. But if it's just a few devices, using a relay is a cheaper solution." />}}
+
+## Relay Requirments
+
+- End device should run the LoRaWAN stack (TS001-LoRaWAN L2 1.0.4 and Regional Parameters RP2-1.0.3 specifications) + LoRa Basics Modem firmware. The experimental release of this firmware can be found at https://github.com/Lora-net/SWL2001
+- End device should be implemented with SX1261/2, LR1110, LR1120, or LR1121 sub-GHz LoRa transceiver.
+- A Network Server supporting the relay specification.
+- The gateways do not need to be updated to support relays.
+
+{{< note "The relay specification requirements are described in the  [LoRaWAN® Relay Specification TS011-1.0.0](https://resources.lora-alliance.org/technical-specifications/ts011-1-0-0-relay) document." />}}
+
+## How Relay Works?
+
+The following steps describe how an end device communicates with the Network Server through a relay.
+
+- The end device that wants to use the support of a relay, first sends a  Wake-on-Radio (WOR) frame. The WOR frame is used to wake up the relay and share the radio parameters like frequency and data rate that will be used in the subsequent uplink frame. There are two types of WOR frames an end device can send to a relay (The type of the WOR frame can be identified by examining its payload content):
+    - Relay Join-Request
+    - Relay Uplink (Class A uplink)
+
+{{< note "The end device and the relay know in advance which channels (frequencies) are used to send/receive WOR/WOR-ACK frames. However, the end device is not yet aware of other radio parameters used by the relay, so it assumes the CAD periodicity is 1 second, the relay's real-time clock accuracy is 40 ppm, and the number of LoRa symbols taken to move from the CAD to the RX state is 8." />}}
```

Comment:
remove note tags

Reply:
I have removed some parts of the above paragraph and moved the rest to the beginning of the section.